aea/protocols/generator.py

```diff
@@ -22,11 +22,11 @@
 import re
 from os import path
 from pathlib import Path
-from typing import Dict, List
+from typing import Dict, Set
 
 from aea.configurations.base import ProtocolSpecification
 
-CUSTOM_TYPE_PATTERN = "[A-Z_][a-zA-Z0-9]*"
+CUSTOM_TYPE_PATTERN = "ct:[A-Z_][a-zA-Z0-9]*"
```

Comment:
In a previous comment I mentioned that this is not the correct regex, please check and replace

Reply:
Done